.dockerignore

```diff
@@ -0,0 +1,4 @@
+.env
+.github
+.vscode
+.pytest
```

Comment:
Not sure what the .pytest folder is. Did you mean .pytest_cache?

.env is the most important, but there are other things you could include, e.g.
```
Dockerfile
.git
.gitignore
geckodriver.exe
```